zebra/cli/l3routes_vty.c

```diff
@@ -1751,7 +1719,7 @@ int route_prefix_comparator(const void *route_entry1,
     /* Caching the strcmp result */
     if (ip1 && ip2) {
         if (is_ipv6_prefix1_valid && is_ipv6_prefix2_valid) {
-            strcmp_result = ipv6_prefix_compare(&ipv6_str1, &ipv6_str2);
+            strcmp_result = IPV6_ADDR_CMP(&ipv6_str1, &ipv6_str2);
```

Comment:
It is a binary, filled with inet_pton on line 1711 and 1715. Don't know why it called ipv6_str.